packages/devtools_app/lib/src/screens/debugger/breakpoints.dart

```diff
@@ -19,28 +17,29 @@ double get executableLineRadius => scaleByFontFactor(1.5);
 double get breakpointRadius => scaleByFontFactor(6.0);
 
 class Breakpoints extends StatefulWidget {
-  const Breakpoints({Key key}) : super(key: key);
+  const Breakpoints({Key? key}) : super(key: key);
 
   @override
   _BreakpointsState createState() => _BreakpointsState();
 }
 
 class _BreakpointsState extends State<Breakpoints> {
-  DebuggerController controller;
+  DebuggerController get controller => _controller!;
+  DebuggerController? _controller;
 
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
 
     final newController = Provider.of<DebuggerController>(context);
     if (newController == controller) return;
```

Comment:
this line needs to be
newController == _controller

to avoid crashing

Reply:
Done